lib/worker.js

```diff
@@ -121,6 +167,8 @@ Worker.prototype.run = function () {
     queue: this.queue,
     job: this.job
   }
+  this._incMonitor('ponos')
+  var timer = monitor.timer('ponos.timer', !process.env.WORKER_MONITOR_DISABLED, this._eventTags())
```

Comment:
> Optional boolean startNow can be provided to start the timer at a later date using the .start method.

The way I'm reading it, if this is set to `true`, then the timer will never be started, but it is created. My worry here is that all the workers will create timers that are never started and they'll all be stored in memory, leading to a memory leak. I would be okay with this block being wrapped with an `if` statement, or wrapping it in a new function like you did with the increment. You'd end up with

```js
Worker prototype.createTimer () { return !process.env.WORKER_MONITOR_DISABLED ? monitor.timer() : null }
 
var timer = this.createTimer()
// ...
if (timer) { timer.stop() }
```